.github/workflows/ci.yml

```diff
@@ -80,18 +80,74 @@ jobs:
           .github/workflows/freebsd-build.sh x86_64
           .github/workflows/ci-run-tests.sh
 
+  build-alpine:
+      runs-on: ubuntu-22.04
+      container:
+        image: alpine:3.20
+        # Needed to attach to a binary
+        options: --security-opt seccomp=unconfined
```

Comment:
Maybe, as you suggested, add this to the instructions somewhere? I guess the docker info can be placed in a `doc/docker.md` file, to reduce the clutter in README.md.